saas/static/js/djaodjin-saas-angular.js

```diff
@@ -917,6 +917,7 @@ metricsControllers.controller("metricsCtrl",
             $scope.ends_at = $scope.ends_at.toDate();
         }
     }
+    $scope.period = 'monthly';
```

Comment:
The angularjs front-end is deprecated though somewhat still working. FYI.

Reply:
Well, that sucks.